app/Http/Livewire/Components/Header.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace App\Http\Livewire\Components;
+
+use App\Http\Livewire\Traits\InteractWithModal;
+use App\Models\Configs;
+use Illuminate\Support\Facades\Config;
+use Livewire\Component;
+
+class Header extends Component
+{
+	use InteractWithModal;
+
+	/**
+	 * @var string
+	 */
+	public $title = '';
+
+	/**
+	 * @var string
+	 */
+	public $mode;
```

Comment:
```suggestion
	public string $mode;
```
Add a PHPdoc comment which explains what kind of mode this is. What are the admissible values? What does each value mean or imply?